ra/ra.go

```diff
@@ -1855,6 +1857,38 @@ func (ra *RegistrationAuthorityImpl) countFailedValidation(ctx context.Context,
 		}
 		ra.log.Warningf("spending against the %s rate limit: %s", ratelimits.FailedAuthorizationsPerDomainPerAccount, err)
 	}
+
+	// Increment ratelimit for IssuancePaused
```

Comment:
Rather than creating a second transaction and making a second call to Spend, this should be bundled into the same transaction and Spend call. That way we don't have to make twice as many calls to Redis.

You can see an multiple rate limit updates being bundled into a single transaction here: https://github.com/letsencrypt/boulder/blob/844334e04aef79d4fabc08aab9b8a95db10611ec/ratelimits/transaction.go#L493

---

Hmm, but now that I say that, I see that processing the result of that bundled call to decide whether to make the PauseIdentifier call would be uglier. Maybe doing it this way is just fine...